### PR TITLE
Login style

### PR DESCRIPTION
**Pull request by mhema42**
I think i´m done with styling of login/registration pages.
Fix error message if wrong user or password.
Fix user can´t register new user if username exist.
Fix active link in navbar for pages beneath main page.

image

image

### CODE REVIEW

File: styles/login.module.css [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+    opacity: 1;
+  }
+}
+
+.error {
+    color: darkred;
+    animation: error 7s;
+}
+
+@keyframes error {
```
Comment (emmaambr) [inline styles/login.module.css:24]: This animation does not seem to work for some reason.

**Review by emmaambr — COMMENTED** [review]
Great job Mats! 👍🏻 
The only thing I noticed is that the error animation does not seem to work (?)

**Comment by mhema42** [comment]
> Great job Mats! 👍🏻 The only thing I noticed is that the error animation does not seem to work (?)

Yes, it seems to work only the first time u enter wrong user/pwd, the next time it shows direkt without animation...
When u reload page it works again, but even then it works a bit strange, but I think it´s becouse of delay, think it works better after run build?

**Comment by emmaambr** [comment]
>

**Comment by Nw84** [comment]
> > 
> 
> tried to reload the page and restart localhost, still does not work for me. But if it works for @Nw84, maybe it's only a temporary problem on my end?

I having the same problems with the animation and i am unsure why 😖  But the otherwise the login page looks great 👍

**Review by Nw84 — APPROVED** [review]
I am having the same problem as Emma with the error message animation and I am unsure what the problem is 😖 But i do not think that it is a big problem and it is okay that it does not animate every time that Richard enters the wrong password 😏   But really good job with the forms and login page Mats, it looks great 👍

**Review by emmaambr — APPROVED** [review]

**Review by mhema42 — COMMENTED** [review]
Tried a litter longer time for the anim on error message, works for me, but it worked with shorter time for me too, strange...?

**Pull request merged**